Controller/AbstractCrudController.php

```diff
@@ -54,44 +50,50 @@
             },
-            'updated_entity_validator' => function(array $params, $mappedEntity, DefaultEntityValidator $defaultValidator, array $config, ContainerInterface $container) {
+            'updated_entity_validator' => function (array $params, $mappedEntity, DefaultEntityValidator $defaultValidator, array $config, ContainerInterface $container) {
                 return $defaultValidator->validate($mappedEntity, $config);
             },
-            'save_entity_handler' => function($entity, array $params, PersistenceHandlerInterface $defaultHandler, ContainerInterface $container) {
+            'save_entity_handler' => function ($entity, array $params, PersistenceHandlerInterface $defaultHandler, ContainerInterface $container) {
                 return $defaultHandler->save($entity);
             },
-            'update_entity_handler' => function($entity, array $params, PersistenceHandlerInterface $defaultHandler, ContainerInterface $container) {
+            'update_entity_handler' => function ($entity, array $params, PersistenceHandlerInterface $defaultHandler, ContainerInterface $container) {
                 return $defaultHandler->update($entity);
             },
-            'batch_update_entities_handler' => function(array $entities, array $params, PersistenceHandlerInterface $defaultHandler, ContainerInterface $container) {
+            'batch_update_entities_handler' => function (array $entities, array $params, PersistenceHandlerInterface $defaultHandler, ContainerInterface $container) {
                 return $defaultHandler->updateBatch($entities);
             },
-            'remove_entities_handler' => function(array $entities, array $params, PersistenceHandlerInterface $defaultHandler, ContainerInterface $container) {
+            'remove_entities_handler' => function (array $entities, array $params, PersistenceHandlerInterface $defaultHandler, ContainerInterface $container) {
                 return $defaultHandler->remove($entities);
             },
-            'exception_handler' => function(\Exception $e, $operation, ExceptionHandlerInterface $defaultHandler, ContainerInterface $container) {
+            'exception_handler' => function (\Exception $e, $operation, ExceptionHandlerInterface $defaultHandler, ContainerInterface $container) {
                 return $defaultHandler->createResponse($e, $operation);
             },
-            'format_new_entity_values' => function(array $params, array $config, NewValuesFactoryInterface $defaultImpl, ContainerInterface $container) {
+            'format_new_entity_values' => function (array $params, array $config, NewValuesFactoryInterface $defaultImpl, ContainerInterface $container) {
                 return $defaultImpl->getValues($params, $config);
             },
+            // allows to override default data mapper used by the this specific controller
+            'create_default_data_mapper' => function(ContainerInterface $container) use ($me) {
```

Comment:
Thanks for running code through php-cs-fixer :)

P.S. One space is missing between function and a brace, please run cf-fixer again.